internal/resources/gitrepository/resource_git_repository_test.go

```diff
@@ -29,25 +29,6 @@ import (
 	testhelper "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/testing"
 )
 
-const (
-	gitRepositoryResource      = ResourceName
-	gitRepositoryResourceVar   = "test_git_repository"
-	gitRepositoryDataSourceVar = "test_data_source_git_repository"
-	gitRepositoryNamePrefix    = "tf-gr-test"
-)
-
-type testAcceptanceConfig struct {
-	Provider                    *schema.Provider
-	GitRepositoryResource       string
-	GitRepositoryResourceVar    string
-	GitRepositoryResourceName   string
-	GitRepositoryName           string
-	ScopeHelperResources        *commonscope.ScopeHelperResources
-	GitRepositoryDataSourceVar  string
-	GitRepositoryDataSourceName string
-	Namespace                   string
-}
-
```

Comment:
@shobha2626 why are we moving this out to the test helper?

Reply:
Reposting the comment - https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/247#discussion_r1311251798

> The functions/variables declared within files consisting of build tags are ignored by the build tool. Since these functions/variables are used at other places also we error saying undefined.
> 
> To overcome this issue we need to move this commonly used functions/variables to a common place. Hence, the change.

Reply:
@shobha2626 have we run all the acceptance tests after making these changes?

Reply:
I have run the acceptances test for majority of the resources and made sure the tags work. Since, there isn't any test changes I don't see any other concern

Reply:
Please run it for source secret, git repository and kustomization resources and ensure they still pass. 

I see test code changes even though they are just changes to the file.

Reply:
Mock test passes for these resources, they make use of the same variables/functions moved to the new file. Hence, I don't see the concern of the test failing due to the changes made in this PR

Reply:
Thanks!